src/wp-includes/html-api/class-wp-html-open-elements.php

```diff
@@ -540,7 +660,16 @@ public function after_element_push( WP_HTML_Token $item ): void {
 		 * cases where the precalculated value needs to change.
 		 */
 		switch ( $item->node_name ) {
```

Comment:
I'm less convinced on these two methods since they aren't spec algorithms and since nothing noticeable from the outside should have changed. before, the HTML Processor shouldn't have allowed getting into any of these tags, but now it does, and things continue to work as expected.

does that make any sense?